ice_in

```diff
@@ -1,450 +1,159 @@
 &setup_nml
-  bfbflag = "off"
+  bfbflag = "off" 
   conserv_check = .false.
-  days_per_year = 365
-  diagfreq = 24
-  dumpfreq = "x"
-  dumpfreq_n = 1
-  hist_avg = .true., .true., .true., .true., .true.
-  histfreq = "m", "x", "x", "x", "x"
-  histfreq_n = 1, 0, 0, 0, 0
-  history_file = "unknown"
-  history_format = "default"
-  history_precision = 4
+  debug_forcing = .true.
+  debug_model = .true.
+  diagfreq = 960
+  dumpfreq = "y"
+  dump_last = .true.
+  histfreq = "d", "m", "x", "x", "x"
   ice_ic = "./input/iced.1900-01-01-10800.nc"
-  latpnt = 90.0, -65.0
   lcdf64 = .true.
-  lonpnt = 0.0, -45.0
-  ndtd = 1
-  numax = 99
-  numin = 11
-  pointer_file = "./rpointer.ice"
-  print_global = .true.
-  print_points = .false.
-  restart_ext = .false.
-  restart_file = ""
-  restart_format = "default"
-  use_leap_years = .false.
-  version_name = "undefined_version_name"
-  write_ic = .false.
-  year_init = 1
+  npt = 35040
+  pointer_file      = './rpointer.ice'
+  print_global      = .false.
 /
 &grid_nml
   bathymetry_file = "./input/topog.nc"
-  bathymetry_format = "default"
-  close_boundaries = .false.
-  dxrect = 30.0e5
-  dyrect = 30.0e5
   grid_atm = "A"
   grid_file = "./input/grid.nc"
   grid_format = "nc"
   grid_ice = "B"
   grid_ocn = "A"
   grid_type = "tripole"
-  gridcpl_file = "unknown_gridcpl_file"
   kcatbound = 0
   kmt_file = "./input/kmt.nc"
-  nblyr = 3
+  nblyr = 1
   ncat = 5
   nfsd = 1
   nilyr = 4
   nslyr = 1
-  orca_halogrid = .false.
-  use_bathymetry = .false.
 /
 &tracer_nml
-  n_aero = 3
-  n_algae = 0
-  n_dic = 0
-  n_doc = 0
-  n_don = 0
-  n_fed = 0
-  n_fep = 0
-  n_iso = 3
-  n_zaero = 0
-  restart_aero = .false.
-  restart_age = .false.
-  restart_fsd = .false.
-  restart_fy = .false.
-  restart_iso = .false.
-  restart_lvl = .false.
-  restart_pond_lvl = .false.
-  restart_pond_topo = .false.
-  restart_snow = .false.
-  tr_aero = .false.
-  tr_fsd = .false.
```

Comment:
`tr_fsd` is required for floe size distribution, we can include it in the next PR